best-practices.md

```diff
@@ -203,6 +222,81 @@ register the media type with IANA, so that other STAC clients can find it easily
+| azimuth | [View Extension](extensions/view/README.md#best-practices) | Points to a file with per-pixel azimuth angles. |
+| sun-azimuth | [View Extension](extensions/view/README.md#best-practices) | Points to a file with per-pixel sun azimuth angles. |
+| sun-elevation | [View Extension](extensions/view/README.md#best-practices) | Points to a file with per-pixel sun elevation angles. |
+| terrain-shadow | [View Extension](extensions/view/README.md#best-practices) | Points to a file that indicates whether a pixel is not directly illuminated due to terrain shadowing. |
+| terrain-occlusion | [View Extension](extensions/view/README.md#best-practices) | Points to a file that indicates whether a pixel is not visible to the sensor due to terrain occlusion during off-nadir viewing. |
+| terrain-illumination | [View Extension](extensions/view/README.md#best-practices) | Points to a file with coefficients used for terrain illumination correction are provided for each pixel. |
+| local-incidence-angle | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the local incidence angle file. |
+| ellipsoid-incidence-angle | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the ellipsoid incidence angle file. |
+| noise-power | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the noise power file. |
+| amplitude | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the intensity file with focused SAR data that has been ground range detected (e.g. GRD). |
+| magnitude | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the intensity file where data are represented as complex numbers containing amplitude and phase information (e.g SLC). |
+| sigma0 | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the radar backscatter file where data is referenced in ground surface. It is often derived from an `amplitude` or a `magnitude` role asset. |
+| beta0 | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the radar backscatter file where data is referenced in the slant range plane and is radiometrically calibrated.  It is often derived from an `amplitude` or a `magnitude` role asset. |
+| gamma0 | [SAR Extension](extensions/sar/README.md#best-practices)  | Points to the radar backscatter file where data is referenced in the plane perpendicular to the line of sight. It is often derived from an `amplitude` or a `magnitude` role asset. |
+| date-offset | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the date-offset file. |
+| covmat | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the Points to the Normalized Polarimetric Radar Covariance Matrix (CovMat) file. |
+| prd | [SAR Extension](extensions/sar/README.md#best-practices) | Points to the Polarimetric Radar Decomposition (PRD) file. |
+
+Some of the particular asset roles also have some best practices.
+
+### Thumbnails
```

Comment:
I thought we agreed on having a chapter that defines recommended sizes and formats for thumbnail, overview and visual? maybe don't say a lot in the table and add a link there to this chapter while extending this chapter with recommendation about thumbnails, overviews etc. in general?

Reply:
Yeah, I started this with the thumbnails, but then I didn't actually know what to recommend for the rest. I think this is the right spot for it.

Reply:
I'll add an issue, and then if someone fills in the actual recommendations on that issue I can do the PR to add them.

Reply:
I thought we had some criteria defined in the last call. Can't exactly recall. Can you, @matthewhanson?